python/src/Pagmo_doc.i.in

```diff
@@ -0,0 +1,246 @@
+| pso                 | Particle Swarm Optimization                     | no              | no      |
++---------------------+-------------------------------------------------+-----------------+---------+
+| pso_gen             | Particle Swarm Optimization Generational        | no              | no      |
++---------------------+-------------------------------------------------+-----------------+---------+
+| sea                 | (N+1)-ES Simple Evolutionary Algorithm          | no              | no      |
++---------------------+-------------------------------------------------+-----------------+---------+
+| sga                 | Simple Genetic Algorithm                        | no              | yes     |
++---------------------+-------------------------------------------------+-----------------+---------+
+| simulated_annealing | Corana’s Simulated Annealing                    | no              | no      |
++---------------------+-------------------------------------------------+-----------------+---------+
+| bee_colony          | Artificial Bee Colony                           | no              | no      |
++---------------------+-------------------------------------------------+-----------------+---------+
+| cmaes               | Covariance Matrix Adaptation Evo. Strategy      | no              | no      |
++---------------------+-------------------------------------------------+-----------------+---------+
+| xnes                | Exponential Evolution Strategies                | no              | no      |
++---------------------+-------------------------------------------------+-----------------+---------+
+| nsga2               | Non-dominated Sorting GA                        | yes             | yes     |
++---------------------+-------------------------------------------------+-----------------+---------+
+| moead               | Multi-objective EA vith Decomposition           | yes             | no      |
++---------------------+-------------------------------------------------+-----------------+---------+
+| mhaco               | Multi-objective Hypervolume-based ACO           | yes             | yes     |
++---------------------+-------------------------------------------------+-----------------+---------+
+| nspso               | Non-dominated Sorting PSO                       | yes             | no      |
++---------------------+-------------------------------------------------+-----------------+---------+
+
+Only gaco and ihs natively support constraints, but for the other algorithms
+constraints are emulated through penalization.
+Also gaco, pso_gen, nsga2, mhaco and nspso support batch evaluation,
+except when constraints are emulated, see :meth:`setBlockSize`.
+Default parameters are available in the :class:`~openturns.ResourceMap` for
+each algorithm, refer to the correspondings keys and the matching documentation in pagmo.
```

Comment:
What is the "matching documentation in pagmo"? Is it this page?

Reply:
it should be pagmo's doc